content/master/concepts/pods.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Crossplane Pods
+weight: 1
+description: Background on the components installed with Crossplane and their functions.
+---
+
+The base Crossplane installation consists of two Pods, the `crossplane` pod and
+the `rbac-manager` pod. Both pods install in the `crossplane-system`
+namespace by default. 
+
+
+## Crossplane pod
+
+### Init pod
+Before starting the core Crossplane pod an _init_ Pod runs. The init pod is
```

Comment:
Why is _init_ `P`od written with a capital here but in lowercase almost everywhere else?